File: benchmarks/string_concat/Strings.java
public class Strings {
    public static void main(String[] args) {
        int N = 100000;

        StringBuilder sb = new StringBuilder();
        for (int i = 0; i < N; i++) {
            sb.append('a');
        }
        System.out.println(sb.length());

        StringBuilder sb2 = new StringBuilder();
        for (int i = 0; i < 10000; i++) {
            sb2.append("abc");
        }
        System.out.println(sb2.length());
    }
}

File: benchmarks/string_concat/strings.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	N := 100000

	var b strings.Builder
	for i := 0; i < N; i++ {
		b.WriteByte('a')
	}
	fmt.Println(len(b.String()))

	var b2 strings.Builder
	for i := 0; i < 10000; i++ {
		b2.WriteString("abc")
	}
	fmt.Println(len(b2.String()))
}
